homeassistant/components/shelly/sensor.py

```diff
@@ -88,20 +88,24 @@ def name(self):
     @property
     def state(self):
         """Value of sensor."""
+        value = getattr(self.block, self.attribute)
+        if value is None:
+            return None
```

Comment:
Is this if statement necessary? Since value will be returned in line 108 after failing to satisfy any of the previous conditions, correct?

Reply:
It's necessary since it can satisfy the condition in line 95.

Reply:
Okay thanks for the clarification :)